cogs/logging.py

```diff
@@ -37,13 +54,64 @@ async def on_message_delete(self,message):
     # TODO implement edit messages
     @commands.Cog.listener()
     async def on_message_edit(self,before,after):
-        pass
+        guild_id = before.guild.id
+        channel = self.logging_channels[guild_id]
+
+        before_string = ("**Before:**" if len(before.content) <= 750
+                        else "**Before:** (Trimmed at 750)")
+        after_string = ("**After:**" if len(after.content) <= 750
+                        else "**After:** (Trimmed at 750)")
```

Comment:
For these we might want to come up with some shared constants, 750 and the 2000 to be used in purge and delete.

Reply:
Delete has to be slightly under 2000 so that we can add metadata to it, such as which user posted each message.

But in general yeah, constants would be really helpful